cc/trees/layer_tree_host.cc

```diff
@@ -1623,6 +1623,9 @@ void LayerTreeHost::UnregisterLayer(Layer* layer) {
   DCHECK(IsMainThread());
   DCHECK(LayerById(layer->id()));
   DCHECK(!in_paint_layer_contents_);
+
+  recordreplay::Assert("[Run-1229-1342] LayerTreeHost::UnregisterLayer %d",
```

Comment:
This should be `RUN-1229-1385` (since that's the child issue its attached to)

Reply:
Ahh, now it finally clicked on what the pattern is... :D